### PR TITLE
feat: update titles for certification pages

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This PR updates the `<title>` for each certification page to include the word "Certification" for SEO purposes.

For example, `Responsive Web Design Certification | freeCodeCamp.org` rather than `Responsive Web Design | freeCodeCamp.org` like we currently have.

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

**Review by raisedadead — COMMENTED** [review]
We should probably add a test for this. Although I am not sure if it is trivial it's to test things with jest when `helmet` is involved.

File: client/src/templates/Introduction/SuperBlockIntro.js [hunk 1]
```diff
@@ -148,11 +148,14 @@ class SuperBlockIntroductionPage extends Component {
     const nodesForSuperBlock = edges.map(({ node }) => node);
     const blockDashedNames = uniq(nodesForSuperBlock.map(({ block }) => block));
     const i18nSuperBlock = t(`intro:${superBlock}.title`);
+    const i18nCertText = t(`intro:misc-text.certification`, {
+      cert: i18nSuperBlock
```
Comment (ShaunSHamilton) [inline client/src/templates/Introduction/SuperBlockIntro.js:152]:
```suggestion
      cert: superBlock
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Should we rather be passing in the `superBlock`, and not the translated `i18SuperBlock`?

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, good point.

I'm not sure how easy it'll be to write Jest tests for Helmet. I'll give it a shot. Otherwise, we should be able to at least do some testing in Cypress.

Speaking of which, it seems like one of the Cypress tests failed. I'll update those now.

File: client/src/templates/Introduction/SuperBlockIntro.js (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply client/src/templates/Introduction/SuperBlockIntro.js:152]: Hey @ShaunSHamilton, thank you for the suggestion.

If I remember correctly, `superBlock` here is the dashed version of the super block, and is only in English (responsive-web-design).

To get the titles to render correctly in all languages, we need to get the localized version of the title first, then pass it as the variable `cert` to `misc-text.certification` in the `intro.json` file.

That should give us `Responsive Web Design Certification | freeCodeCamp.org`, `Certificación de Diseño Web Responsivo | freeCodeCamp.org`, and so on.

**Comment by raisedadead** [comment]
> Just let me know if it would be better to test with Jest here and I can circle back around.

I will defer to you on whatever is the easiest. All I care about is we do not introduce a regression later. And having any sort of testing in place is good.

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, okay, sounds good.

Right now I'm thinking that Cypress would be easier here. And hopefully later we'll figure out a good way to test the different localization in Cypress later, so we can ensure the titles are being set correctly.

File: client/src/templates/Introduction/SuperBlockIntro.js (hunk 1 above)
Comment (ShaunSHamilton) [reply client/src/templates/Introduction/SuperBlockIntro.js:152]: Right. My bad, I forgot we used an object to pass variables 🤦 , and not to search for properties.

Comment (scissorsneedfoodtoo) [reply client/src/templates/Introduction/SuperBlockIntro.js:152]: No worries! i18next is a whole other layer of complexity on top over everything else we've got going on. Thanks again for taking a look at this.

**Comment by scissorsneedfoodtoo** [comment]
Are there any other changes that should be made @ShaunSHamilton? Otherwise I think this should be ready for another quick round of review.

Note that I just made some minor changes to the current Cypress tests we have. I figure that should be enough to show us whether the new title is rendered correctly.

**Review by ShaunSHamilton — DISMISSED** [review]
This LGTM 👍 

---

Unrelated to this PR, but noticed, and want to make sure:
- The _Coding Interview Prep_ section also comes up as _"Coding Interview Prep Certification"_ , but it is not a certification.

Is this ok?

**Comment by scissorsneedfoodtoo** [comment]
@ShaunSHamilton, that's a great point! Thanks for catching that.

I think you're right, we should probably filter that out since it's not an actual certification. I'll make a quick update to this to make sure we don't add the word "Certification" to that section.

**Review by ShaunSHamilton — APPROVED** [review]
This makes sense. Thanks, Kris.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**